src/ensembl/src/content/app/browser/BrowserCog.tsx

```diff
@@ -22,6 +24,11 @@ const BrowserCog: FunctionComponent<BrowserCogProps> = (
 ) => {
   const { cogActivated, updateSelectedCog, index } = props;
 
+  const ref = useRef(null);
+  useOutsideClick(ref, () => {
+    updateSelectedCog('');
+  });
```

Comment:
Since this is a one statement function how about writing it as `() => updateSelectedCog('')`

Reply:
That's done